src/.travis.yml.jinja

```diff
@@ -6,7 +6,11 @@ cache:
 
 python:
   {#- Pin the lowest supported version for each Odoo release #}
+  {%- if odoo_version < 15 %}
   - "3.6"
+  {%- else %}
+  - "3.8"
```

Comment:
Hm. [Not so long ago](https://github.com/odoo/odoo/pull/70398#issuecomment-832601798) Odoo SA's runbot was still -on 3.6. Did that change recently ?

As of now, Odoo 15.0's setup.py still says >= 3.6

The latest "official" [statement](https://github.com/odoo/odoo/issues/76148#issuecomment-916081840) I know of was that it would >= 3.7 but it was not put into action ?

Don't get me wrong, I'd love to be able to use Python 3.8 features and libraries, but on the other hand, publishing OCA addons that are not compatible with some Odoo deployment is not something we should do lightly.

Reply:
We know that odoo says some things and does others. For me, the runbot logs is the most trustable source of truth. Check them out, they're 3.8. I think that we should just test in the same version as Odoo or we'll have different bugs, which is not fun. And after all we have to make a decision and we cannot defer it anymore. 🤷‍♂️